pgmpy/tests/test_inference/test_ExactInference.py

```diff
@@ -86,6 +86,14 @@ def test_map_query_with_evidence(self):
                                                       {'J': 0, 'Q': 1, 'G': 0})
         self.assertDictEqual(map_query, {'A': 1, 'R': 0, 'L': 0})
 
+    def test_induced_graph(self):
+        induced_graph = self.bayesian_inference.induced_graph(['G', 'Q', 'A', 'J', 'L', 'R'])
+        result_edges = sorted([sorted(x) for x in induced_graph.edges()])
```

Comment:
This isn't exactly what I want to do. The induced graph is undirected, so I sort each edge in order to get (u, v) with u <= v.